packages/percy-storybook/src/cli.js

```diff
@@ -66,13 +74,19 @@ export async function run(argv) {
   // debug('assets %o', assets);
 
   const stories = await getStories(assets[storybookJavascriptPath], options);
-  debug('stories %o', stories);
+  options.debug && debug('stories %o', stories);
```

Comment:
This toggling of output should be the job of the `debug` function itself, I believe you can just do `debug.enabled = options.debug` to toggle it on or off. https://www.npmjs.com/package/debug#checking-whether-a-debug-target-is-enabled